src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -767,7 +767,9 @@ function calc_gradient_interface_flux!(surface_flux_values,
             u_ll, u_rr = get_surface_node_vars(cache_parabolic.interfaces.u,
                                                equations, dg, i,
                                                interface)
-            flux = 0.5f0 * (u_ll + u_rr) # Bassi-Rebay 1 (BR1)
+
+            flux = flux_parabolic(u_ll, u_rr, Gradient(),
+                                  mesh, equations, parabolic_scheme)
```

Comment:
Since this is only used internally, it is fine to keep it as it is right now. When we generalize it to other mesh types, I guess we also need to pass the `orientation` (or `normal_direction`) to allow the same minimal dissipation LDG method on curved meshes.

Reply:
Yes, though I am not sure of the best way to treat topologically non-Cartesian meshes yet. 

Usually we would determine the LDG switch using a fake "upwinding" vector `v0` based on `sign(dot(v0, normal))`, but I am worried about the case when `v0` is nearly parallel to `normal` and the dot product is close to machine zero. 

I hope to address this soon after generalizing to `StructuredMesh` and adaptive meshes.